spec/controllers/admin_outgoing_message_controller_spec.rb

```diff
@@ -55,6 +55,36 @@
 
     end
 
+    context 'unsuccessfully destroying the message' do
```

Comment:
The latter. ISWYM, but I don't think it really implies "destruction occurred but that was undesirable". Any suggestions?